01-arch/07-reputation.md

```diff
@@ -2,6 +2,11 @@
 
 NeoFS reputation system is a subsystem for calculating trust in a node. It is based on a reputation model for assessing trust, which is, in turn, based on the EigenTrust algorithm designed for peer-to-peer reputation management. The algorithm ensures that there is a uniquely defined manager (parent node) for each network participant at each specified time (epoch). Based on the information received from its child node and other managers, it iteratively puts a complex general (Global) Trust of the entire network into the applicable child node.
 
+The reputation system makes it possible to determine the metric of the quality of the nodes (trust). In a situation when most of the nodes make an honest assessment of the actions of other nodes, the system allows you to calculate this metric quite accurately. This metric can be used for:
+
+1. Filtering the list of nodes where the user intends to store information. For example, use the services of only nodes with a score higher than acceptable minimum for the user.
```

Comment:
For example, one can use only those nodes whose scores are higher than the minimum acceptable for the user.

Reply:
fixed